src/test/java/com/redhat/devtools/lsp4ij/LSPIJUtils_toOffsetTest.java

```diff
@@ -8,6 +8,7 @@
  * Contributors:
  * Red Hat, Inc. - initial API and implementation
  ******************************************************************************/
+

```

Comment:
Whitespace-only changes after moving the tests for this change into their own test class below.